examples/otlptrace/src/main/java/com/google/cloud/opentelemetry/example/otlptrace/OTLPTraceExample.java

```diff
@@ -49,6 +49,7 @@ private static OpenTelemetrySdk setupTraceExporter() throws IOException {
   }
 
   // Modifies the span exporter initially auto-configured using environment variables
+  // Note: This adds static authorization headers which are set only at initialization time.
```

Comment:
```suggestion
  // Note: This adds static authorization headers which are set only at initialization time.
  // This will stop working after the token expires, since the token is not refreshed.
```